api/src/main/java/org/apache/iceberg/actions/MigrateTable.java

```diff
@@ -50,6 +50,16 @@ default MigrateTable dropBackup() {
     throw new UnsupportedOperationException("Dropping a backup is not supported");
   }
 
+  /**
+   * Sets a table name for the backup of the original table
```

Comment:
Minor: Missing `.` at the end of the sentence?

Reply:
Sorry for this. Add `.` at the end of the sentence in a next commit.